README.rst

```diff
@@ -1,84 +1,138 @@
+2. Open your AWS console. 
+3. In the landing page, under **Compute**, click **Lambda**.
+4. Click **Create function** to create a layer with SignalFx's capabilities.
+5. Click **Author from scratch**.
+6. In **Function name**, enter a descriptive name for the wrapper. 
+7. In **Runtime**, select the desired language.
+8. Click **Create function**. 
+9. Click on **Layers**, then add a layer.
+10. Mark **Provide a layer version**.
+11. Enter an ARN number. 
+  * To locate the ARN number, see [Lambda Layer Versions](https://github.com/signalfx/lambda-layer-versions/blob/master/python/PYTHON.md).
 
-**Note: the environment variables ``SIGNALFX_INGEST_ENDPOINT`` and ``SIGNALFX_AUTH_TOKEN`` are being deprecated and will not be supported in future releases.**
+~~~~~
 
-.. code:: bash
+**Option 3: Create a Lambda function, then create and attach a layer based on a SignalFx template**
 
-    SIGNALFX_ACCESS_TOKEN=access token
+In this option, you will chose a SignalFx template, and then deploy a copy of the layer.
+
+1. Open your AWS console. 
+2. In the landing page, under **Compute**, click **Lambda**.
+3. Click **Create function** to create a layer with SignalFx's capabilities.
+4. Click **Browse serverless app repository**.
+5. Click **Public applications**.
+6. In the search field, enter and select **signalfx-lambda-python-wrapper**.
+7. Review the template, permissions, licenses, and then click **Deploy**.
+    * A copy of the layer will now be deployed into your account.
+8. Return to the previous screen to add a layer to the function, select from list of runtime compatible layers, and then select the name of the copy.  
+
+
+~~~~~
+
+Step 2: Update the SignalFx realm
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, this function wrapper will send data to the ``us0`` realm. If you are not in this realm, and you are not using the gateway, then you will need to update the realm for the SignalFx subdomain, such as us1.signalfx.com ( {REALM}.signalfx.com  ). 
+
+To locate your realm:
+
+1. Open SignalFx and in the top, right corner, click your profile icon.
+2. Click **My Profile**.
+3. Next to **Organizations**, review the listed realm.
+
+Additionally, you must adjust the variables you set to the correct realm ingest endpoint (``https://ingest.{REALM}.signalfx.com``). These variables include ``SIGNALFX_ENDPOINT_URL``, ``SIGNALFX_TRACING_URL``, ``SIGNALFX_METRICS_URL``, and ``SIGNALFX_INGEST_ENDPOINT``. 
 
-    # endpoint for both metrics and tracer. Overridden by SIGNALFX_METRICS_URL
-    # and SIGNALFX_TRACING_URL if set
-    SIGNALFX_ENDPOINT_URL=http://<my_gateway>:8080
 
-    # optional metrics and tracing configuration
+~~~~~
 
+Step 3: Set environment variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Based on your realm, you may need to set specific environment variables. 
+  * Regardless of your realm, every user must set SIGNALFX_ACCESS_TOKEN. 
+  * If you are not in the us0 realm, then you must set SIGNALFX_ENDPOINT_URL. 
```

Comment:
What would you say about: 

If you are using pops, gateway or wish to use ingest directly from realm other than us0, you must set at least one of endpoint variables appropriately (SIGNALFX_ENDPOINT_URL or SIGNALFX_METRICS_URL or SIGNALFX_INGEST_ENDPOINT (deprecated)). See below. 

?

Reply:
I'm a little confused with SIGNALFX_INGEST_ENDPOINT. Is it already deprecated? If so, should we even mention it to the user? 

Please review the changes to this section. I wasn't sure if your comments were meant to replace the rest of the content in that section. 

-----
(Removed content)
You can use ``SIGNALFX_ENDPOINT_URL`` to configure a common endpoint for metrics and
traces. In other words, you can set ``SIGNALFX_ENDPOINT_URL`` to replace ``SIGNALFX_TRACING_URL`` and ``SIGNALFX_METRICS_URL``. However, if you explicitly set either ``SIGNALFX_TRACING_URL`` or ``SIGNALFX_METRICS_URL``, then ``SIGNALFX_ENDPOINT_URL`` will be overwritten.  

Review the following examples. 

If only ``SIGNALFX_ENDPOINT_URL`` is set, then both metrics and traces will be sent to the gateway address:

.. code:: bash

    SIGNALFX_ENDPOINT_URL=http://<my_gateway>:8080


If both ``SIGNALFX_ENDPOINT_URL`` and ``SIGNALFX_METRICS_URL`` are set, then traces will be sent to the gateway and metrics will go through POPS.

.. code:: bash
    
    SIGNALFX_ENDPOINT_URL=http://<my_gateway>:8080
    SIGNALFX_METRICS_URL=https://pops.signalfx.com

Reply:
I only meant to replace one sentence "Based on your realm, you may need... " and the two options below :) 
SIGNALFX_INGEST_ENDPOINT can still be used, but a user will get the deprecation warning if they set it. I only wanted to leave it here in case anyone who has existing deployments/config comes back to the doc for some answers.

Reply:
Based on what I thought was your feedback, versus your real feedback, versus what I edit, please let me know if my new "step 3" makes sense. If not, I can easily revert to the original step 3.